target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8065-r7800.dts

```diff
@@ -165,6 +180,13 @@
 					output-high;
 				};
 			};
+
+			usbvcc_pins: usbvcc_pins {
+				pins = "gpio15", "gpio16", "gpio68";
+				function = "gpio";
+				drive-strength = <12>;
+				bias-pull-down;
```

Comment:
Erm really thanks for switching to readable values, but I meant output-high property as on line 180 :)